blueprints/app/files/package.json

```diff
@@ -12,6 +12,8 @@
   "repository": "",
   "scripts": {
     "build": "ember build",
+    "lint": "eslint *.js addon app config lib node-tests server tests",
+    "lint:fix": "<% if (yarn) { %>yarn run lint --fix<% } else { %>npm run lint -- --fix<% } %>",
```

Comment:
for yarn we might not need `lint:fix` since `yarn lint --fix` will just work out of the box

given that template lint and stylelint also exist maybe we should call it `eslint` instead of `lint`? 🤔